cmake/patches/flatbuffers/flatbuffers_cmake.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 3987eac..5e5462f 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -223,7 +223,7 @@ elseif(CMAKE_COMPILER_IS_GNUCXX)
+       "${CMAKE_CXX_FLAGS} -std=c++0x")
+   endif(CYGWIN)
+   set(CMAKE_CXX_FLAGS
+-    "${CMAKE_CXX_FLAGS} -Wall -pedantic -Werror -Wextra -Werror=shadow")
++    "${CMAKE_CXX_FLAGS} -Wall -pedantic -Werror -Wextra -Werror=shadow -Wno-error=stringop-overflow")
```

Comment:
You can also pass the new flag through command line instead of patching flatbuffer.  Like:

```
CFLAGS="-O2 -g -Wno-error=stringop-overflow" CXXFLAGS="-O2 -g -Wno-error=stringop-overflow" cmake ${ORT_SOURCE_DIR)\cmake 
```

 Maintaining a patch could be costly.